bot/code_review_bot/report/base.py

```diff
@@ -152,14 +151,17 @@ def pluralize(word, nb):
 
         # Build top comment
         nb = len(issues)
-        extras = ""
 
-        comment = COMMENT_FAILURE.format(
-            extras_comments=extras,
-            defects_total=pluralize("defect", nb),
-            diff_id=revision.diff_id,
-            defects="\n".join(defects),
-        )
+        if nb > 0:
+            comment = COMMENT_FAILURE.format(
+                defects_total=pluralize("defect", nb), diff_id=revision.diff_id
+            )
+        else:
+            comment = ""
+
+        # Add defects
+        if defects:
+            comment += "\n".join(defects) + "\n"
```

Comment:
Before returning the comment at the end of the function, could you assert that it is not empty as a safe-guard?

Reply:
Done